src/components/navbarbutton.tsx

```diff
@@ -0,0 +1,19 @@
+import React,{useState} from 'react'
+import Select, { Option } from 'react-button-navbar';
+ 
+
+
+    function App(){
```

Comment:
Hmm, extra tabs? Please align the function to left.

Reply:
Can I add this code?
headline: {
    textAlign: 'center', // <-- the magic
    fontWeight: 'bold',
    fontSize: 18,
    marginTop: 0,
    width: 200,
    backgroundColor: 'brown',
  }